apps/costcontrol/js/app.js

```diff
@@ -411,26 +405,24 @@ var CostControlApp = (function() {
     return window.location.hash.split('#')[1] === 'datausage-tab';
   }
 
-  function startFTE() {
-    window.addEventListener('message', function handler_finished(e) {
-      if (e.origin !== Common.COST_CONTROL_APP) {
-        return;
-      }
-
-      var type = e.data.type;
-
-      if (type === 'fte_finished') {
-        window.removeEventListener('message', handler_finished);
-        document.getElementById('splash_section').
-          hidden = 'true';
-
-        // Only hide the FTE view when everything in the UI is ready
-        ConfigManager.requestAll(function() {
-          startApp(Common.closeFTE);
-        });
-      }
-    });
+  function _handler_finished(e) {
```

Comment:
This is a legacy name, rename to follow camel case convenion and provide a more meaningful name.